HearthWatcher/FriendlyChallengeDialogVisibilityWatcher.cs

```diff
@@ -0,0 +1,54 @@
+﻿using HearthWatcher.EventArgs;
+using HearthWatcher.Providers;
+using System;
+using System.Threading.Tasks;
+
+namespace HearthWatcher
+{
+	public class FriendlyChallengeDialogVisibilityWatcher
```

Comment:
Can we call this just "FriendlyChallengeWatcher"? :P

Reply:
Agreed :)